src/pl/wkos/textstats/TextStats.java

```diff
@@ -0,0 +1,60 @@
+package pl.wkos.textstats;
+
+public class TextStats {
+    public static int numberOfSignsWithoutWhiteSpaces(String text) {
+        int counter = 0;
+        for (int i = 0; i < text.length(); i++) {
+            char sign = text.charAt(i);
+            if (!Character.isWhitespace(sign)) counter++;
```

Comment:
Jak robisz ifa to staraj się zawsze dawać nawiasy klamrowe mimo że w niektórych przypadkach (jak ten) nie musisz.

Reply:
Poprawione.